plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -175,6 +175,9 @@ open class KtlintPlugin : Plugin<Project> {
             // Android option is available from ktlint 0.12.0
             runArgs.add("--android")
         }
+        if (extension.ruleSets.size > 0) {
+            extension.ruleSets.forEach { runArgs.add("--ruleset=$it") }
```

Comment:
@Tapchicoma Shouldn't there be a version check on this so that if the ruleset is configured on an older version, this will fail?

Reply:
@JLLeitschuh check one of my previous PR review comments - this option is available since initial KtLint release.

Reply:
Okay, awesome.